http/client_tx.go

```diff
@@ -26,6 +26,14 @@ type Transaction struct {
 	http *httpClient
 }
 
+func NewTransaction(rawURL string, id uint64) (*Transaction, error) {
```

Comment:
This was added so the CLI tests have access to the transactions. This should be cleaned up when we switch to an internal transaction pool.